tests/lightning_qubit/test_adjoint_jacobian_class.py

```diff
@@ -497,7 +504,11 @@ def test_hermitian_tensor_expectation(self, tol, lightning_sv):
                 qml.expval(qml.Hermitian(obs, wires=(0,)) @ qml.PauliZ(wires=1))
             tape.trainable_params = {0}
 
-            statevector.reset_state()
+            if device_name == "lightning.gpu":
+                statevector.reset_state(True)
```

Comment:
Idem

Reply:
Do you suggest removing the option of `sync`?
Probably we can propose a new way to define a "global" variable for `sync` or is it better just to remove it? :thinking:

Reply:
Solved!